paramak/neutronics_utils.py

```diff
@@ -119,3 +124,223 @@ def add_stl_to_moab_core(
+            results[tally.name]['MeV per source particle'] = {
+                'result': tally_result / 1e6,
+                'std. dev.': tally_std_dev / 1e6,
+            }
+
+            if fusion_power is not None:
+                results[tally.name]['Watts'] = {
+                    'result': tally_result * 1.602176487e-19 * (fusion_power / ((17.58 * 1e6) / 6.2415090744e18)),
+                    'std. dev.': tally_std_dev * 1.602176487e-19 * (fusion_power / ((17.58 * 1e6) / 6.2415090744e18)),
+                }
+
+        if tally.name.endswith('flux'):
+
+            data_frame = tally.get_pandas_dataframe()
+            tally_result = data_frame["mean"].sum()
+            tally_std_dev = data_frame['std. dev.'].sum()
+            results[tally.name]['Flux per source particle'] = {
+                'result': tally_result,
+                'std. dev.': tally_std_dev,
+            }
+
+        if tally.name.endswith('spectra'):
+            data_frame = tally.get_pandas_dataframe()
+            tally_result = data_frame["mean"]
+            tally_std_dev = data_frame['std. dev.']
+            results[tally.name]['Flux per source particle'] = {
+                'energy': openmc.mgxs.GROUP_STRUCTURES['CCFE-709'].tolist(),
+                'result': tally_result.tolist(),
+                'std. dev.': tally_std_dev.tolist(),
+            }
+
+        if tally.name.startswith('tritium_production_on_2D_mesh'):
+
+            _save_2d_mesh_tally_as_png(
+                score='(n,Xt)',
+                tally=tally,
+                filename='tritium_production_on_2D_mesh' + tally.name[-3:]
+            )
+
+        if tally.name.startswith('flux_on_2D_mesh'):
+
+            _save_2d_mesh_tally_as_png(
+                score='flux',
+                tally=tally,
+                filename='flux_on_2D_mesh' + tally.name[-3:]
+            )
+
+        if tally.name.startswith('heating_on_2D_mesh'):
+
+            _save_2d_mesh_tally_as_png(
+                score='heating',
+                tally=tally,
+                filename='heating_on_2D_mesh' + tally.name[-3:]
+            )
+
+        if '_on_3D_mesh' in tally.name:
+            mesh_id = 1
+            mesh = statepoint.meshes[mesh_id]
+
+            xs = np.linspace(
+                mesh.lower_left[0],
+                mesh.upper_right[0],
+                mesh.dimension[0] + 1
+            )
+            ys = np.linspace(
+                mesh.lower_left[1],
+                mesh.upper_right[1],
+                mesh.dimension[1] + 1
+            )
+            zs = np.linspace(
+                mesh.lower_left[2],
+                mesh.upper_right[2],
+                mesh.dimension[2] + 1
+            )
+            tally = statepoint.get_tally(name=tally.name)
+
+            data = tally.mean[:, 0, 0]
+            error = tally.std_dev[:, 0, 0]
+
+            data = data.tolist()
+            error = error.tolist()
+
+           for content in [data, error]:
```

Comment:
```suggestion
            for content in [data, error]:
```

There's a missing space here causing a bad indentation. Maybe that's my bad...

Reply:
ah I think i spotted that when fixing the other things and then just noticed this message